docs/sources/panels-visualizations/query-transform-data/transform-data/index.md

```diff
@@ -967,10 +967,10 @@ Here is the result after adding a Limit transformation with a value of '3':
 ### Time series to table transform
 
 {{% admonition type="note" %}}
-This transformation is available in Grafana 9.5+ as an opt-in beta feature. Modify Grafana [configuration file]({{< relref "../../../setup-grafana/configure-grafana/#configuration-file-location" >}}) to enable the `timeSeriesTable` [feature toggle]({{< relref "../../../setup-grafana/configure-grafana/#feature_toggles" >}}) to use it.
+This transformation is available in Grafana 9.5+ as an opt-in beta feature. Modify Grafana [configuration file][] to enable the `timeSeriesTable` [feature toggle][] to use it.
```

Comment:
Hmm, there's a few of these. I'm gonna have to spend some time thinking about how we can make reference style links work in shortcodes before we can merge this PR.